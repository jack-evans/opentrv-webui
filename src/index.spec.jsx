
describe('index.jsx', () => {
  it('renders without crashing', () => {
    require('./index')
  })
})
