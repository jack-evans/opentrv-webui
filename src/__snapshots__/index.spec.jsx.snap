// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.jsx renders without crashing 1`] = `"{\\"props\\":{\\"children\\":{\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}}},\\"context\\":{},\\"refs\\":{},\\"updater\\":{},\\"history\\":{\\"length\\":1,\\"action\\":\\"POP\\",\\"location\\":{\\"pathname\\":\\"/\\",\\"search\\":\\"\\",\\"hash\\":\\"\\"}},\\"_reactInternalInstance\\":\\"censored\\",\\"state\\":null}"`;
